packages/matchbox/src/components/TextField/TextField.js

```diff
@@ -30,7 +30,7 @@ const FieldBox = props => {
       borderRadius="100"
       bg={props.disabled ? 'gray.200' : 'white'}
       lineHeight="2.5rem"
-      height="2.5rem"
+      height={props.height}
```

Comment:
I don't think we previously supported height, I think you might be able to just remove this line. 

_or_, if you wanted to support system props, you'll need to modify this one line
```js
import { margin, height } from 'styled-system';

const componentProps = omit(rest, margin.propNames, height.propNames);
```

Reply:
Just saw this below – nevermind, I see where props.height comes from
```
const inputProps = {
    ...componentProps,
    as: multiline ? 'textarea' : 'input',
    height: multiline ? 'auto' : '2.5rem',
```